src/components/Plans/PlanCalculator/Discounts/index.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { SUBSCRIPTION_TYPE } from '../../../../doppler-types';
+import { useIntl } from 'react-intl';
+
+export const Discounts = ({ discounts, selectedDiscount, onSelectDiscount }) => {
+  const intl = useIntl();
+  const _ = (id, values) => intl.formatMessage({ id: id }, values);
+
+  const getDiscountName = (subscriptionType) => {
+    switch (subscriptionType) {
+      case SUBSCRIPTION_TYPE.monthly:
+      case SUBSCRIPTION_TYPE.quarterly:
+      case SUBSCRIPTION_TYPE.biyearly:
+      case SUBSCRIPTION_TYPE.yearly:
+        return _('plan_calculator.discount_' + subscriptionType.replace('-', '_'));
+      default:
+        return '';
+    }
```

Comment:
Es necesario el switch? por que no simplemente usar la función?

```suggestion
    switch (subscriptionType) {
        return _('plan_calculator.discount_' + subscriptionType.replace('-', '_'));
    }
```

Si el valor de `subscriptionType` puede no ser válido, creo que debería estar validado antes.

Reply:
Si, está validado antes. 
Quitaré el switch